timesketch/lib/testlib.py

```diff
@@ -191,6 +211,34 @@ def create_index(self, *args, **kwargs):
         """Mock creating an index."""
         return
 
+    def import_event(self, index_name, event_type, event=None,
+                     event_id=None, flush_interval=None):
+        """Mock adding the event to Elasticsearch, instead add the event
+        to event_store.
+
+        Args:
+            flush_interval: Number of events to queue up before indexing. (This
+            functionality is not supported.)
+            index_name: Name of the index in Elasticsearch
+            event_type: Type of event (e.g. plaso_event)
+            event: Event dictionary
```

Comment:
I would call this event_dict

Reply:
It's also called event in import_event in elastic.py (the function we are mocking). Should we leave this, or change both?

Reply:
Ah, that's right. Then leave it :)